src/streaming/constants/Constants.js

```diff
@@ -332,6 +339,9 @@ export default {
     ID3_SCHEME_ID_URI: 'https://aomedia.org/emsg/ID3',
     COMMON_ACCESS_TOKEN_HEADER: 'common-access-token',
     DASH_ROLE_SCHEME_ID : 'urn:mpeg:dash:role:2011',
+
+    ENHANCEMENT_CODECS: ['lvc1'],
```

Comment:
With this code, 'lvc1' gets always considered as supported by src/streaming/utils/Capabilities.js#L141
However, shouldn't we reject this 4CC if lcevc_dec.js is not registered as extension?

Reply:
Thank you, agreed! I would like to modify Capabilities.js#L141 to also check if lcevc_dec.js is loaded, and reject otherwise.

May I ask for clarification regarding `registered as extension` - is there a way in dash.js to register lcevc_dec.js as a loaded extension? I could check if `window.LCEVCdec` is defined but I am wondering if there is a more idiomatic way.